### PR TITLE
Update v4 guide, add section about new namespaces

### PR DESCRIPTION
**Pull request by Zyphrax**
- A few small layout changes to keep it in style with the other docs
- Added a section about the new namespaces
- Mention of the v4 upgrade guide in the main README

### CODE REVIEW

File: docs/v4-upgrade.md [hunk 1]
```diff
@@ -87,8 +94,24 @@ nodeModulesLoader.exclude.push(/some-library/) // replace `some-library` with
...
+
+# After
+"runtime~hello_react" => "/packs/js/runtime~hello_react-da2baf7fd07b0e8b6d17.js"
+```
+
+## Example Upgrades
 
 This is what an upgrade to Webpacker 4 looked like for existing Rails apps (please contribute yours!):
 
-- https://github.com/connorshea/ContinueFromCheckpoint/pull/77
```
Comment (connorshea) [inline docs/v4-upgrade.md]: Why remove the list syntax here?

Comment (Zyphrax) [reply docs/v4-upgrade.md]: I did a few markup changes based on the MD linter in VS Code. Not sure how I removed the list syntax, but I've corrected it 🙂

**Comment by codebryo** [comment]
Changes look amazing. Thanks for the clarity!

**Comment by gauravtiwari** [comment]
Thanks @Zyphrax Could you please rebase with master?

**Comment by Zyphrax** [comment]
@gauravtiwari Done!

**Comment by manju345** [comment]
`/money：Exchange`

**Pull request merged**